src/main/java/utils/Parser.java

```diff
@@ -0,0 +1,79 @@
+package utils;
+
+import java.time.LocalDate;
+
+import commands.*;
+import exceptions.InvalidInputException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+
+public class Parser {
+
+    /**
+     * Return command according to user input
+     */
+    public static Command parse(String command) throws InvalidInputException {
+        String[] commands = command.split(" ", 2);
+        command = commands[0];
+
+        String description = commands.length > 1 ? commands[1] : "";
+        
+        if (command.equals("bye")) {
```

Comment:
Could consider using a switch-case statement for greater clarity

Reply:
I agree! Will add alongside enums in future commits.